test/SourceKit/VariableType/fully-qualified.swift

```diff
@@ -0,0 +1,35 @@
+let x: Int = 3
```

Comment:
This is just a copy of `basic.swift`, right? You could also just add another `RUN:` line to it with `%FileCheck %s --check-prefix CHECK-FULLY-QUALIFIED`

Reply:
It is indeed, I was looking for a way to do this, but missed the custom prefix. Thanks for pointing it out!